src/ripple/ledger/OpenView.h

```diff
@@ -51,9 +54,11 @@ class OpenView
 
     // List of tx, key order
     using txs_map = std::map<key_type,
-        std::pair<std::shared_ptr<
-            Serializer const>, std::shared_ptr<
-                Serializer const>>>;
+        std::pair<std::shared_ptr<Serializer const>,
+        std::shared_ptr<Serializer const>>,
+        std::less<key_type>, qalloc_type<std::pair<key_type const,
+        std::pair<std::shared_ptr<Serializer const>,
+        std::shared_ptr<Serializer const>>>, false>>;
```

Comment:
@HowardHinnant I'm impressed. The reason I broke out `txs_map_mapped_type` (definitely the more correct name) the way I did is because it's pretty complicated by itself, so replacing it twice with a label makes the `txs_map` spec a lot clearer to read. IMHO, breaking out the second type doesn't buy us as much in terms of removing `<>`s. :)

...But while we're going crazy:
```
using txs_map_base = std::map<key_type, std::pair<std::shared_ptr<Serializer const>,
    std::shared_ptr<Serializer const>>>;
using txs_map = std::map<txs_map_base::key_type, txs_map_base::mapped_type,
    std::less<txs_map_base::key_type>,
    qalloc_type<txs_map_base::value_type, false>>;
```
LULZ.

Ok, now that I've wasted everybody's time, is it worth making any of the changes, or better to just leave it as-is?